Add security-council doc and updated 2 docs (#75)
maticnetwork/zkevm-docs: docs/protocol/security-council.md

---
id: security-council
title: Security Council
sidebar_label: Security Council
description: This document provides an explanation of the Security Council Multisig of the Polygon zkEVM.
keywords:
  - polygon
  - zkEVM
  - zkNode
  - deploy full zkevm
  - governance
  - contract
---

> In addition to the above-mentioned governance issues and security measures, one more component was essential, especially to a young L2 system such as the Polygon zkEVM. That component is, the _Security Council Multisig_.

Reviewer: ```suggestion
In addition to the previously mentioned governance issues and security measures, one more component was essential, especially to a young L2 system such as the Polygon zkEVM. That component is, the _Security Council Multisig_.
```

- [0xBDc2…FEFf](https://etherscan.io/address/0xBDc235cC9d6Baa641c5ae306bc83962475A5FEFf#code)
- [0x4c16…8891](https://etherscan.io/address/0x4c1665d6651ecEfa59B9B3041951608468b18891#code)
- [0x3ab9…D622](https://etherscan.io/address/0x3ab9f4b964eE665F7CDf1d65f1cEEc6196B0D622#code)
- [0x49c1…0E86](https://etherscan.io/address/0x49c15936864690bCd6af0ecaca8E874adFF30E86#code)
- [0x9F7d…86A0](https://etherscan.io/address/0x9F7dfAb2222A473284205cdDF08a677726d786A0#code)
- [0x2188…1C28](https://etherscan.io/address/0x21887c89368bf918346c62460e0c339113801C28#code)

### Security Council Multisig?

The Security Council Multisig is a _multisig contract_ deployed by the Polygon zkEVM Security Council when either an _emergency state_ is triggered or an _emergency upgrade_ needs to be executed.

The _multisig contract_ is a _6-out-of-8_ multisig, which requires six (6) signatures of the Security Council to be attached for the contract to be successfully deployed.

There is a further stipulation that a minimum of 2 out of the 6 attached signatures must be from among the 4 members who are external to Polygon.

## Conclusion

> Although the ultimate goal is to move towards a totally decentralised Polygon zkEVM, employing a security council multisig is inevitable for the early stages of the zkrollup.

Reviewer: ```suggestion
Although the ultimate goal is to move towards a totally decentralized Polygon zkEVM, employing a security council multisig is inevitable for the early stages of the zkRollup.
```

> It is a trade-off between security and decentralisation. So then, for the sake of the long-term security, it is a deliberate decision to have more centralised early stages of development, in order to attain more decentralised later stages.

Reviewer: ```suggestion
It is a trade-off between security and decentralization. So then, for the sake of long-term security, it is a deliberate decision to have more centralized early stages of development, in order to attain more decentralized later stages.
```
